__mocks__/@celo/identity/index.ts

```diff
@@ -21,15 +21,11 @@ enum AuthenticationMethod {
 
 export const OdisUtils = {
   Query: {
-    ODIS_ALFAJORESSTAGING_CONTEXT: {
-      odisUrl: 'alfajoresstaging',
-      odisPubKey: 'alfajoresstaging',
-    },
-    ODIS_ALFAJORES_CONTEXT: {
+    ODIS_ALFAJORES_CONTEXT_PNP: {
       odisUrl: 'alfajores',
       odisPubKey: 'alfajores',
     },
-    ODIS_MAINNET_CONTEXT: {
+    ODIS_MAINNET_CONTEXT_PNP: {
```

Comment:
i did verify that this is the context that returns the values that we were using before